.travis.yml

```diff
@@ -1,9 +1,13 @@
 language: node_js
 node_js: ["0.10"]
+cache:
+  directories:
+    - node_modules
+    - bower
 before_install:
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
 before_script:
-  - npm install -g bower gulp
```

Comment:
I saw you switched to use the local bower (which is great 👍 ) Travis will use the `test` script in package.json to run the tests themselves: https://github.com/CrossLead/angular-zendesk-widget/blob/397a7c4a6398a8cf68cd7ea8e6fe9450da34c567/package.json#L9 which is still using `gulp process-all` and will fail if we need to clear the cache (as there will no longer be a global gulp install)